Tasks/Maslov/task4/Mtask4.py

```diff
@@ -0,0 +1,22 @@
+
+with open('text.py.txt', 'r+', encoding='utf-8') as f:
+    s = f.readlines()
+    print(s)
+    
+a = int(input('Введите максимальное количество символов в строке ,минимальное 35 :  '))
+b = ""
+c = 0
+for i in s:
+    for j in i.split():
+        d = c + len(j)
+        if c != 0:
+            d += 1
+        if d > a:
+            b += '\n'
+            c = 0
+        if c != 0:
+            b += ' '
+            c += 1
+        b += j
+        c += len(j)
+    print(b)
```

Comment:
basically, it does not work properly

first of all, you need to add some additional spaces to every line in order for the text to be spread evenly

secondly, you need to write the result in a new file

thirdly, you've specified a minimal value for a user, but did nothing to check the value

and in the end you supposed to write the result in a new file, it should look like this:
![image](https://user-images.githubusercontent.com/57627287/153128707-95d308ba-26f7-45fd-8a31-64e9cccddba9.png)

whereas your output looks like this:
![image](https://user-images.githubusercontent.com/57627287/153128835-cd6a2425-e57f-4fe9-b4ea-d5cc4ec28194.png)
(the text should not have such stepped right side)